### ISSUE TITLE
Barely visible input placeholder in Firefox

### ISSUE THREAD

**Issue by vyachkonovalov** [jgthms/bulma#2886]
Bulma CSS framework, bug
Bulma version 0.8.1
Firefox Developer Edition 75.0b7

Description:
Firefox default `opacity: 0.5` for input/textarea placeholders makes them barely visible.

**Comment by nainardev** [comment]
Can you provide a screenshot?

**Comment by vyachkonovalov** [comment]
Sure
![screen](https://i.imgur.com/iNcyMIL.png)

> Bulma version 0.8.1

Oops, 0.8.1 is good, the problem only in master branch.

**Comment by jgthms** [comment]
Probably same issue as #2885

**Comment by vyachkonovalov** [comment]
Just tested with revert of 5ba407bf91107fb6284327d23253065caccd9604, the problem still exists. (here's a page I use https://github.com/vyachkonovalov/bulma_issue_2886)

**Comment by ondrej-tucek** [comment]
Hi, I don't want to open a new issue because it's maybe the same error. If I open bulma.io -> [form](https://bulma.io/documentation/form/input/) with FF v74 (left screenshot), the color of placeholder is different than in Chromium v80 (right).

![compare](https://user-images.githubusercontent.com/13625279/77820989-c3eb7b00-70e6-11ea-8f55-908d31a350fe.png)

**Comment by surister** [comment]
Yeah, placeholder opacity is almost invisible in Firefox, maybe make this modifiable via variables?
![image](https://user-images.githubusercontent.com/37985796/78277842-6fe0fc00-7515-11ea-9dc9-bc6a04fae3eb.png)

**Comment by scheinercc** [comment]
I would say, to make it cross browser compatible the RGBA value, which doesn't work for Firefox, should be changed to a RGB + an opacity value instead, this will give the same result in the browsers I quickly checked:

https://github.com/jgthms/bulma/blob/master/sass/form/shared.sass#L19
https://github.com/jgthms/bulma/blob/master/sass/form/shared.sass#L52


    $input-disabled-placeholder-color: $input-disabled-color !default
    $input-disabled-placeholder-opacity: 0.3 !default
    ...

        +placeholder
              color: $input-disabled-placeholder-color
              opacity: $input-disabled-placeholder-opacity

**Comment by scheinercc** [comment]
@jgthms what do you think of my suggestion above? RGB + opacity works in all browsers, while RGBA values only in some.

**Comment by rukshn** [comment]
I can also confirm that the placeholders are barely visible on FireFox, please fix in coming release @jgthms

**Comment by terabytesoftw** [comment]
![image](https://user-images.githubusercontent.com/42547589/103485065-6bae7b80-4dd2-11eb-9c91-1ed93173efe3.png)

- That capture is Chrome, as seen in FireFox, Edge, the color of the placeHolder must be corrected.

**Comment by getup8** [comment]
As a workaround, set the placeholder color variable to whatever color you actually want it to be:
```scss
$input-placeholder-color: #999;
```
And then make sure Firefox shows it at 100% opacity:
```scss
::placeholder {
  opacity: 1;  // For Firefox.
}
```

I do think it'd make sense to add that `opacity` setting to bulma to account for Firefox.

**Comment by scheinercc** [comment]
@jgthms you mind having a quick look. It's a pretty small change. Cheers
